misc/docs/source/packages/obspy.signal.rst

```diff
@@ -9,7 +9,7 @@
        :toctree: autogen
        :nosignatures:
 
-       ~array_analysis.array_processing
+       ~array_analysis._covariance_array_processing
```

Comment:
This probably doesn't need to be here any more, but I don't really know how the docs are built.